modules/bitgo/src/v2/keychains.ts

```diff
@@ -380,6 +380,9 @@ export class Keychains {
     }
 
     const mpcUtils = new MpcUtils(this.bitgo, this.baseCoin);
-    return await mpcUtils.createKeychains({ passphrase: params.passphrase });
+    return await mpcUtils.createKeychains({
+      passphrase: params.passphrase,
+      originalPasscodeEncryptionCode: params.originalPasscodeEncryptionCode,
+    });
```

Comment:
while you're here, could you pass in `params.enterprise` too?

Reply:
done